### PR TITLE
Add mini_pupper_control package

### PR DESCRIPTION
**Pull request by Ericsii**
- Add `mini_pupper_control` package for ROS Humble

### CODE REVIEW

**Comment by Ericsii** [comment]
Previous PR #35

File: README.md [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-rosdep install --from-paths src --ignore-src -r -y
 cd ..
-colcon build
-sudo apt-get install ros-galactic-teleop-twist-keyboard ros-galactic-cartographer-ros
+rosdep install --from-paths src --ignore-src -r -y
+colcon build --symlink-install
+sudo apt-get install ros-humble-teleop-twist-keyboard ros-humble-cartographer-ros
 ```
 
 ## 2. Test in RVIZ
```
Comment (Tiryoh) [inline README.md]:
```suggestion
## 2. Test in RViz
```

File: mini_pupper_control/mini_pupper_control/servo_interface.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# @Author  : Zhengxiao Han
+
+import numpy as np
+import rclpy
+from rclpy.node import Node
+from trajectory_msgs.msg import JointTrajectory
+from MangDang.mini_pupper.HardwareInterface import HardwareInterface
+
+
+class SERVO_INTERFACE(Node):
```
Comment (Tiryoh) [inline mini_pupper_control/mini_pupper_control/servo_interface.py]: How about using CamelCase?

https://peps.python.org/pep-0008/#class-names

```suggestion
class ServoInterface(Node):
```

File: mini_pupper_control/mini_pupper_control/servo_interface.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            [rf2_position, lf2_position, rb2_position, lb2_position],
+            [rf2_position + rf3_position, lf2_position + lf3_position,
+             rb2_position + rb3_position, lb2_position + lb3_position]
+        ])
+        self.hardware_interface.set_actuator_postions(joint_angles)
+
+
+def main(args=None):
+    rclpy.init(args=args)
+    servo_interface_node = SERVO_INTERFACE()
```
Comment (Tiryoh) [inline mini_pupper_control/mini_pupper_control/servo_interface.py]:
```suggestion
    servo_interface_node = ServoInterface()
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,28 @@
...
+cd ros2_ws/src
 git clone https://github.com/mangdangroboticsclub/mini_pupper_ros.git -b ros2
 git clone --recursive https://github.com/chvmp/champ -b ros2
-rosdep install --from-paths src --ignore-src -r -y
 cd ..
-colcon build
-sudo apt-get install ros-galactic-teleop-twist-keyboard ros-galactic-cartographer-ros
+rosdep install --from-paths src --ignore-src -r -y
+colcon build --symlink-install
+sudo apt-get install ros-humble-teleop-twist-keyboard ros-humble-cartographer-ros
```
Comment (Tiryoh) [inline README.md]: This is just a note for future changes.
This time we are updating only the mini_pupper_control package, so this is fine, but let's change it when we update the "mini_pupper_navigation" package.
We can add cartographer to `package.xml` in "mini_pupper_navigation" and remove the `apt-get install ros-humble-cartographer-ros` command in README.

File: README.md [hunk 5]
```diff
@@ -1,29 +1,28 @@
 # Mini Pupper ROS2
 
 ## 1. Installation
-Ubuntu 20.04 + ROS2 Galatic is required.
+Ubuntu 22.04 + ROS2 Humble is required. Please follow the [installation document for ROS Humble](https://docs.ros.org/en/humble/)
```
Comment (Tiryoh) [inline README.md]:
```suggestion
Ubuntu 22.04 + ROS2 Humble is required. Please follow the [installation document for ROS Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
```

**Review by Tiryoh — COMMENTED** [review]
Tested with my real Mini Pupper and confirmed it workes with the following command :+1:

```sh
# 1st terminal
ros2 launch mini_pupper_bringup bringup.launch.py
# 2nd terminal
ros2 launch mini_pupper_control servo_interface.launch.py
# 3rd terminal
ros2 run teleop_twist_keyboard teleop_twist_keyboard
```

@Ericsii Could you add pytest scripts for Python format checking like this?
https://github.com/ros2/examples/tree/humble/rclpy/topics/minimal_publisher/test
https://github.com/ros2/examples/blob/humble/rclpy/topics/minimal_publisher/package.xml#L15-L20

File: README.md (hunk 4 above)
Comment (Ericsii) [reply README.md]: README is updated

Comment (Tiryoh) [reply README.md]: Thanks!

**Review by Tiryoh — APPROVED** [review]

**Pull request merged**